src/components/Post/Post.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC, useState } from "react";
+// import styles from "./Users.module.css";
+import Post from "../../Models/Post";
+import { Anchor } from "@mantine/core";
+import { useParams } from "react-router-dom";
+
+const PostItem: FC<Post> = (props: Post) => {
+  const [post] = useState<Post>(props);
+  let showUser = (
+    <div>
+      <Anchor href={`/posts/${props.id}`}>{props.text}</Anchor>
+    </div>
+  );
+
+  const { postsId } = useParams();
+  if (postsId == post.id) return showUser;
+  if (postsId == undefined) return showUser;
```

Comment:
Do you want me to explain what I did here?

Reply:
Yeah, doesn't make sense

Reply:
I only want to render the user which id is as in the url.

Reply:
If the userId is undefined than it will render showUser for all users.
If the userId is equal to the user id in the url it will only render it once for said userId.

Reply:
Never mind that
I changed it and removed this entirely :)